src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Automation/Peers/ComboBoxAutomationPeer.cs

```diff
@@ -51,16 +51,20 @@ override protected string GetClassNameCore()
         override public object GetPattern(PatternInterface pattern)
         {
             object iface = null;
+            ComboBox owner = (ComboBox)Owner;
 
             if (pattern == PatternInterface.Value)
             {
-                ComboBox owner = (ComboBox)Owner;
                 if (owner.IsEditable) iface = this;
             }
             else if(pattern == PatternInterface.ExpandCollapse)
             {
                 iface = this;
             }
+            else if (pattern == PatternInterface.Scroll)
+            {
+                if (owner.IsDropDownOpen) iface = this;
```

Comment:
You shouldn't just return "this" here, you should be calling the base method, because it does not return "this" it returns the AutomationPeer for the ScrollHost. Short-circuit when the drop-down is closed, but still call the base method when it is open.

https://github.com/dotnet/wpf/blob/8c7ab1592b6c70e732106aeabf58f215d4b71c22/src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Automation/Peers/ItemsControlAutomationPeer.cs#L36-L59